lib/wibox/layout/flex.lua

```diff
@@ -50,14 +50,35 @@ local flex = {}
 -- @name insert
 -- @class function
 
+--- The widget used to fill the spacing between the layout elements.
+--
+-- By default, no widget is used.
+--
+--@DOC_wibox_layout_flex_spacing_widget_EXAMPLE@
+--
+-- @property spacing_widget
+-- @param widget
+
+--- Add spacing between each layout widgets.
+--
+--@DOC_wibox_layout_flex_spacing_EXAMPLE@
+--
+-- @property spacing
+-- @tparam number spacing Spacing between widgets.
```

Comment:
Why is this not in `@DOC_fixed_COMMON@`?

Reply:
The examples are not the same